website/site/content/download/personal-edition/ubuntu.md

```diff
@@ -147,6 +147,44 @@ Change the dbconfig setting to use the postgres database you created:
 "dbconfig": "postgres://boardsuser:boardsuser-password@localhost/boards?sslmode=disable&connect_timeout=10",
 ```
 
+## Install MySQL 
+
+Alternate to Postgress you also can store your data in a MySQL/MariaDB database.
+To install, run:
+
+```
+sudo apt install mariadb-server mariadb-client
+```
+
+Log in as root in your database:
+```
+sudo mysql 
+```
+
+On the MySQL prompt, run the following commands (**change the user/password** to your own values):
+<pre>
```

Comment:
I'm not familiar with `<pre>` - is it typed like that in the command prompt/terminal?

Reply:
no, it's bad HTML 
should be replaced with ```

Reply:
this `<pre>` is still not replaced.